nextjs-legacy/README.md

```diff
@@ -0,0 +1,52 @@
+This is an example of an instrumented Next.js app.
+
+## Overview
+
+This example application shows a list of users and provides a form for editing
+individual users. When editing a user's name, no error will occur. When editing
+a user's age, an error condition will be encountered.
+
+A few source files to pay particular attention to:
+
++ [next.config.js](./next.config.js): shows configuration necessary to get
+instrumentation for all of the libraries New Relic's Node.js agent supports.
++ [newrelic.js](./newrelic.js): shows how to enable application logging. This
```

Comment:
logging is on by default. I would just remove this

Reply:
One of the criteria for this app was to show logging. I think that means it should be specifically called out. How about a comment in the `newrelic.js` file that says "the configuration is enabled by default, but listed here to provide an example."?

Reply:
sure. i was just calling out that configuration is not needed, it's on by default and i doubt we'll ever change that behavior

Reply:
Comment added. Please take a look.